### ISSUE TITLE
Feature Request: Social Auth

### ISSUE THREAD

**Issue by Nneji123** [seapagan/fastapi-template#332]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to use social accounts for authentication(GitHub, twitter, Google etc). Is it possible to integrate that into this project?

**Comment by seapagan** [comment]
Yes, it is definitely possible and I'll add it to the TODO when I get back to my dev machine, thanks for the suggestion. It will be after I complete the migration to SQLAlchemy which should be soon and then any other pressing bits.

There is a package that is written exactly for this [fastapi-sso](https://github.com/tomasvotava/fastapi-sso) (and probably others) which is probably where to start.

**Comment by Nneji123** [comment]
Thank you very much I'll take look at the package you mentioned.
